docs/apis-tools/migration-manuals/migrate-to-camunda-api.md

```diff
@@ -0,0 +1,95 @@
+- Naming, response codes, and type handling have been streamlined for all endpoints to provide a consistent UX.
+- Endpoints with similar concerns (variable search, for example) have been consolidated into single endpoints.
+
+**Changes for endpoints that query for data:**
+
+- Unified search request structure.
+  - Attributes `filter`, `page`, and `sort` on root level.
+  - Endpoint-specific filter attributes in the filter object, not at the root level.
+  - Pagination information in the `page` object. For example, the attributes `from`, `limit`, `searchBefore`, and `searchAfter`.
+  - Sorting configuration in sort object array, each object containing the field name and order (descending or ascending).
+- Unified search response structure.
+  - Attributes `items` and `page` on root level.
+  - List of endpoint-specific response items in `items` attribute.
+  - Page information in `page` attribute, for example the attributes `totalItems`, `firstSortValues`, and `lastSortValues` to use in `searchBefore` and `searchAfter` in follow-up requests.
+
+<!--- TBD since currently in progress: Filter attributes can use Advanced Search capabilities depending on their type. TBD: We need to see how much of this we manage to implement with 8.7 (which endpoints, which attributes, which types). --->
+
+### Name changes and mappings
+
+The following conventions apply to all attributes:
+
+- `key` and `id` fields contain the entity as a prefix, for example, `userTaskKey`, `processDefinitionId`. This applies when referencing other resources like `formKey` in the user task entity, in the respective entities themselves like `userTaskKey` in the user task entity.
+- The full entity is the prefix to avoid confusion, for example `processDefinitionKey` instead of `processKey` (the latter could be interpreted as process instance or process definition).
+- Other attributes of entities themselves have no prefix to avoid clutter, for example version in the process definition entity. In other resources, they have to be referenced with a prefix, like `processDefinitionVersion` in the process instance entity.
+- The `bpmnProcessId` is now called `processDefinitionId` to be easily relatable to the entity (process definition) and the accompanying `processDefinitionKey`.
+- The `decisionKey` and `dmnDecisionKey` are now aligned to `decisionDefinitionKey`, the `decisionId` and `dmnDecisionId` to `decisionDefinitionId`. Similar to the `processDefinitionId`, those attributes are now related to the entity `decisionDefinition`.
+
+### Operate
+
+<V1 endpoint>
+V2 endpoint to use:
+Input adjustments:
+<change>
+
+Output adjustments
+<change>
+
+### Tasklist
+
+Search Tasks - POST /v1/tasks/search
+V2 endpoint to use: POST /v2/user-tasks/search
+
+Input adjustments:
+
+- Filter attribute `assigned (boolean)` removed
+  - Use filter attribute `assignee` with condition `{ “$exists”: false }`
```

Comment:
🔧 I think this is one typical Google Doc copy-paste issue I have encountered in my own docs PRs over the past years 😄 
```suggestion
  - Use filter attribute `assignee` with condition `{ "$exists": false }`
```